tests/atf_python/atf-python.3

```diff
@@ -0,0 +1,156 @@
+.\" Copyright (c) 2025 The FreeBSD Foundation.
```

Comment:
Unless this is a FreeBSD Foundation sponsored work, you can use your name as the copyright owner.